docs/DeveloperGuide.md

```diff
@@ -320,6 +320,9 @@ The first 2 operations are exposed in the `Model` interface as `Model#canUndo()`
 
 Given below is an example usage scenario and how the undo mechanism behaves at each step.
 
+<div markdown="span" class="alert alert-info">
+:information_source: **Note:** The `ModelManager**` in the following diagrams is actually stored as `Pair&ltMcGymmy&ltPair&ltPredicate&ltFood&gt, MacroList&gt&gt`
+</div>
```

Comment:
Within the `<div></div>` I think u need to use HTML syntax rather than md syntax. 
so ``` ` should be replaced with <code></code>```